datastream-common/src/main/java/com/linkedin/datastream/common/BrooklinEnvelope.java

```diff
@@ -11,7 +11,6 @@
 
 import org.apache.avro.reflect.Nullable;
 import org.apache.commons.lang.Validate;
-import org.apache.kafka.common.header.Headers;
```

Comment:
Now that this import is removed from here, can you check if the Kafka dependency can be removed from `datastream-common` module? IIRC Headers was the only dependency.

Reply:
removed, but had to add it to datastream-utils since it was transitively getting kafka via datastream common haha